ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -1862,10 +1862,11 @@ let rec perform_atomic ~progress_callback ?subtask:_ ?result (op : atomic)
           (Xenops_task.id_of_handle t)
           (List.length atoms) description
       in
+      let with_tracing = parallel_id_with_tracing parallel_id t in
```

Comment:
This name is confusing, as `with_x` normally is a function, while this is a string.